docs/api.md

```diff
@@ -2410,13 +2397,36 @@ Bear in mind that if the first click of the `dblclick()` triggers a navigation e
 
 > **NOTE** `elementHandle.dblclick()` dispatches two `click` events and a single `dblclick` event.
 
-#### elementHandle.fill(value)
+#### elementHandle.fill(value[, options])
 - `value` <[string]> Value to set for the `<input>`, `<textarea>` or `[contenteditable]` element.
+- `options` <[Object]>
+  - `clear` <[boolean]> Whether to clear existing value before typing the new one. Defaults to `true`.
```

Comment:
I think that fill should always clear. Not clearing the input and filling it should almost always be typing. And if they really need it, they can call sendCharacters themselves.